package.json

```diff
@@ -75,7 +75,7 @@
     "react-dom": "16.8.1",
     "react-native-web": "0.11.2"
   },
-  "main": "dist/index.js",
+  "main": "src/index.js",
```

Comment:
Why this change was necessary?

Reply:
@brunohkbx This is going to be removed before merging. The issue Adam is having is that he's pointing his project to his forked version which doesn't contain `dist` since it's `.gitignore`.